compiler-cli/src/publish.rs

```diff
@@ -123,6 +124,53 @@ fn check_for_name_squatting(package: &Package) -> Result<(), Error> {
     Ok(())
 }
 
+fn check_for_multiple_top_level_modules(package: &Package, i_am_sure: bool) -> Result<(), Error> {
+    // Collect top-level module names
+    let mut top_level_module_names = package
+        .modules
+        .iter()
+        .filter_map(|module| module.name.split('/').next())
+        .collect::<Vec<_>>();
+
+    // Remove duplicates
+    top_level_module_names.sort_unstable();
+    top_level_module_names.dedup();
+
+    // If more than one top-level module name is found, prompt for confirmation
+    if top_level_module_names.len() > 1 {
+        println!(
+            "Your package defines multiple top-level modules: {}",
+            top_level_module_names.join(", ")
+        );
+        println!(
+            "Defining multiple top-level modules can lead to namespace pollution \
+and potential conflicts for consumers."
+        );
+        println!();
+        println!(
+            "To fix this, move all your modules under a single top-level module of your choice."
+        );
+        println!();
+        println!("For example:");
+        println!(
+            "  src/{}.gleam\n  src/{}/module1.gleam\n  src/{}/module2.gleam",
+            &package.config.name, &package.config.name, &package.config.name
+        );
+        println!();
```

Comment:
Could you use a multi-line string rather than multiple println statements, and call the `wrap` function to ensure the lines are not too long please